utils/config.go

```diff
@@ -36,3 +36,25 @@ func MustLoadConfig() types.Config {
 
 	return config
 }
+
+func ParseGRPCEndpoint(configEndpoint string, networkName string) (*types.Network, error) {
```

Comment:
I would not expose this to other code. Isn't it better to parse endpoints on load?